javascript/modules/client/src/internal/interfaces/client.ts

```diff
@@ -14,7 +14,9 @@ export interface IClient extends IClientCore {
       tokenAddresses: string[],
     ) => Promise<AssetBalance[]>;
     /** Retrieves the list of transfers from or to the given DAO, by default, ETH, DAI, USDC and USDT on Mainnet*/
-    getTransfers: (daoAddressOrEns: string) => Promise<IAssetTransfers>;
+    // TODO
+    // return typing
+    getTransfers: (daoAddressOrEns: string, searchParams: ISearchParams) => Promise<any>;
```

Comment:
Fabrice proposed a `SerarchParams` interface i the last commit, using the graphql like search params

```ts
export type DaoQueryOptions = {
  sortBy?: DaoSortBy;
  sortDirection?: "asc" | "desc";
  skip?: number;
  limit?: number;
};
```

Maybe changing this to something like this would make more sense, let me know what you think: 

```ts
export interface ISearchParams {
  limit?: number
  skip?: number
  sortDirection?: "asc" | "desc"
}

export interface ITransferSearchParams  extends ISearchParams{
  sortBy?: TransferSortBy
  addressOrEns?: string
}

export interface IDaoSearchParams  extends ISearchParams{
  sortBy?: DaoSortBy
  addressOrEns?: string
}
```
@brickpop

Reply:
The proposal below makes sense, if so I'd propose different namings:
- `IPagination` with `limit`, `skip` and `sortDirection`
- Anything extending it, named like `ITransferQueryParams`, `IDaoQueryParams`, ... `xxxQueryParams`